pedrovgs/HaveANiceDay
# Configure flyway

### :pushpin: References
* **Issue:** Support DB migrations.

### :tophat: What is the goal?

Configure [flyway](https://flywaydb.org) to be able to apply DB migrations easily.

### How is it being implemented?

We've configured flyway sbt plugin and tried a first migration adding a table named ``developers``. The db connection used in the sbt plugin points at the local db. The documentation has been updated showing how to use ``flyway CLI`` pointing at a different DB.

### How can it be tested?

Checkout this branch, start a local MYSql DB named ``haveaniceday`` using ``haveaniceday``  as credentials and execute ``sbt flywayMigrate``.  The migration should be performed properly and a new table named ``developer`` should appear now with some data inside.
